Source/ee/EeExecutor.cpp

```diff
@@ -41,7 +41,10 @@ CEeExecutor::CEeExecutor(CMIPS& context, uint8* ram)
 
 CEeExecutor::~CEeExecutor()
 {
-
+#if defined(__APPLE__)
```

Comment:
Just to stay consistent with other platforms, I think this cleanup code should be moved in `RemoveExceptionHandler`.

Reply:
Sorry for taking this long to get back to this.

I've just checked, moving this to `RemoveExceptionHandler` still causes the same crash, most likely since the class is deconstructed before the thread is stopped.